docs/standard/modernize-with-azure-and-containers/modernize-existing-apps-to-cloud-optimized/when-to-deploy-windows-containers-to-azure-container-service-kubernetes.md

```diff
@@ -1,20 +1,22 @@
 ---
 title: When to deploy Windows Containers to Azure Container Service (that is, Kubernetes)
-description: .NET Microservices Architecture for Containerized .NET Applications | When to deploy Windows Containers to Azure Container Service (that is, Kubernetes)
+description: Modernize existing .NET applications with Azure Cloud and Windows containers | When to deploy Windows Containers to Azure Container Service (that is, Kubernetes)
 author: CESARDELATORRE
 ms.author: wiwagn
-ms.date: 10/26/2017
+ms.date: 04/30/2018
 ---
 # When to deploy Windows Containers to Azure Container Service (that is, Kubernetes)
 
 Azure Container Service optimizes the configuration of popular open-source tools and technologies specifically for Azure. You get an open solution that offers portability both for your containers and for your application configuration. You select the size, the number of hosts, and the orchestrator tools. Azure Container Service handles the infrastructure for you.
 
 If you are already working with open-source orchestrators like Kubernetes, Docker Swarm, or DC/OS, you don't need to change your existing management practices to move container workloads to the cloud. Use the application management tools that you're already familiar with, and connect via the standard API endpoints for the orchestrator of your choice.
 
-All these orchestrators are mature environments if you are using Linux Docker containers, but they also support Windows Containers as of 2017 (some earlier, some more recently, depending on the orchestrator).
+All these orchestrators are mature environments if you are using Linux Docker containers, but might only be in Preview state for Windows Containers.
 
-For example, in Kubernetes, support for containers is native (first-class citizen), so using Windows Containers on Kubernetes is also very effective and reliable (in preview until early fall 2017).
+For example, in Kubernetes, support for containers is native (first-class citizen), so using Windows Containers on Kubernetes is also effective (in preview in ACS as of early 2018).
+
+Important note: The evolved and “more PaaS” version of ACS (Azure Container Service) for Kubernetes is AKS (Azure Kubernetes Service), however, Windows Containers are still not supported as of Q2 2018, but it will be supported soon.
```

Comment:
Not supposed to use ACS acronym for Azure Container Service (https://worldready.cloudapp.net/Styleguide/Read?id=2696&topicid=40742). After first mention, you can just say Container Service.

OK to use AKS for the other one but it should be introduced as Azure Kubernetes Service (AKS). However, since this is the end of the text, you don't really need to introduce the acronym here.